src/masternodes/order.h

```diff
@@ -0,0 +1,222 @@
+#ifndef DEFI_MASTERNODES_ORDER_H
+#define DEFI_MASTERNODES_ORDER_H
+
+#include <amount.h>
+#include <uint256.h>
+
+#include <flushablestorage.h>
+#include <masternodes/res.h>
+
+class COrder
+{
+public:
+    static const int DEFAULT_ORDER_EXPIRY = 2880;
+
+    //! basic properties
+    std::string ownerAddress;
+    std::string tokenFrom;
+    std::string tokenTo;
```

Comment:
Why need to add `tokenFrom` and `tokenTo` if already have the idTokenFrom and idTokenTo?

Reply:
I can probably remove it. This is legacy from my first iteration where I stored only token symbols but then I decided to store the id and I left symbols just so that I don't have to fetch the token to display the symbol, but I'll probably remove it to reduce size in db.